pkg/controller/case_test.go

```diff
@@ -934,6 +958,670 @@ func (ct *controllerTest) v1Now() *metav1.Time {
 	return &n
 }
 
+// TimeoutError simulates timeout error in provision ServiceInstance test
+type TimeoutError string
+
+// Timeout method require for TimeoutError type to meet the url/timeout interface
+func (e TimeoutError) Timeout() bool {
+	return true
+}
+
+// Error returns the TimeoutError as a string
+func (e TimeoutError) Error() string {
+	return string(e)
+}
+
+// SetupEmptyPlanListForOSBClient sets up fake OSB client response to return plans which not exist in any ServiceInstance
+func (ct *controllerTest) SetupEmptyPlanListForOSBClient() {
+	ct.fakeOSBClient.CatalogReaction.(*fakeosb.CatalogReaction).Response = &osb.CatalogResponse{
+		Services: []osb.Service{
+			{
+				Name:        testClusterServiceClassName,
+				ID:          testClassExternalID,
+				Description: "a test service",
+				Bindable:    true,
+				Plans: []osb.Plan{
+					{
+						Name:        "randomPlan",
+						Free:        truePtr(),
+						ID:          "randomID",
+						Description: "This is plan which should not exist in any of instance",
+					},
+				},
+			},
+		},
+	}
+}
+
+// WaitForInstanceCondition waits until ServiceInstance `status.conditions` field value is equal to condition in parameters
+// returns error if the time limit has been reached
+func (ct *controllerTest) WaitForInstanceCondition(condition v1beta1.ServiceInstanceCondition) error {
```

Comment:
Is there a  reason you duplicated this instead of using the one in test/util/util.go?